modules/programs/thunderbird.nix

```diff
@@ -7,6 +7,15 @@ let
 
   cfg = config.programs.thunderbird;
 
+  authMethods = {
+    password = 3;
+    encrypted-password = 4;
+    gssapi = 5;
+    ntlm = 6;
+    tls = 7;
+    oauth2 = 10;
+  };
```

Comment:
Really? What happened to 1, 2, 8 and 9?

Reply:
I was surprised too, but those are the methods accessible from the UI. I guess the missings IDs got deprecated and removed from the drop-down list at some point?